.github/workflows/build.yml

```diff
@@ -118,13 +118,33 @@ jobs:
           - ruby: '3.4'
             gemfile: openssl_2_1
             os: windows-latest
+          - ruby: '2.4'
+            os: ubuntu-20.04
```

Comment:
We had to disable `ruby-2.4` due to some certificate errors when connecting to RubyGems:

```
ERROR:  SSL verification error at depth 1: unable to get local issuer certificate (20)
ERROR:  You must add /OU=GlobalSign Root CA - R3/O=GlobalSign/CN=GlobalSign to your local trusted store
ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
    SSL_connect returned=1 errno=0 state=error: certificate verify failed (https://api.rubygems.org/specs.4.[8](https://github.com/cedarcode/tpm-key_attestation/actions/runs/14000043374/job/39204243539#step:11:9).gz)
```

May come back to this in the future to see if I can fix it.